style.css

```
@@ -0,0 +1,643 @@
+    background-position: bottom -22.82px right -141px;
+}
+
+.background::after {
+    content: '';
+    background-image: url('./img/knights_and_pawn.png');
+    background-size: cover;
+    width: 485.8px;
+    height: 268.11px;
+    position: absolute;
+    bottom: -50.7px;
+    right: 44.55px;
+}
+
+header {
+    position: relative;
+    padding: 26px 72px 0 72px;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 23.4px;
+}
+
+main-navigation {
+    margin: 26px auto 62px 0;
+    padding: 0;
+    display: flex;
+}
+
+.logo {
+    display: block;
+    width: 245.17px;
+}
+
+img {
+    display: block;
+    margin: 0;
+    padding: 0;
+}
+
+main {
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 24px;
+}
+
+main a {
+    font-size: 20px;
+    line-height: 24px;
+    text-decoration: none;
+    color: rgba(48, 87, 162, 1);
+}
+
+section {
+    position: relative;
+    padding-left: 72px;
+    padding-right: 72px;
+}
+
+.preview {
+    padding-top: 61px;
+    padding-bottom: 130px;
+}
+
+.preview-text {
+    margin: 0;
+    padding: 0;
+    width: 542px;
+
+}
+
+.announce p {
+    margin: 0;
+    padding: 0;
+    font-family: Merriweather;
+    font-size: 60px;
+    line-height: 66px;
+    text-align: left;
+    text-transform: uppercase;
+}
+
+.announce .center-text {
+    display: block;
+    text-align: center;
+}
+
+.announce .right-text {
+    display: block;
+    text-align: right;
+}
+
+.announce-text {
+    font-weight: 400;
+    font-family: Golos Text;
+    font-size: 18px;
+    line-height: 23.4px;
+    text-align: center;
+    margin: 24px auto 40px;
+}
+
+.links {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: space-between;
+}
+
+
+.links a {
+    font-family: Golos Text;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 20.8px;
+    display: flex;
+    align-items: center;
+    width: 263px;
+    height: 72px;
+    box-sizing: border-box;
+    padding: 20px 49px;
+    border-radius: 66px;
+    text-decoration: none;
+    text-align: center;
+}
+
+.link-about {
+    color: rgba(31, 31, 31, 1);
+    border: 2px solid rgba(31, 31, 31, 1);
+}
+
+.link-about:hover {
+    background-color: rgba(31, 31, 31, 1);
+    color: rgba(255, 255, 255, 1);
+    border: none;
+}
+
+.link-support {
+    background-color: rgba(31, 31, 31, 1);
+    color: rgba(255, 255, 255, 1);
+}
+
+.link-support:hover {
+    background-color: rgba(251, 206, 81, 1);
+    color: rgba(31, 31, 31, 1);
+}
+
+.support-tournament {
+    padding-top: 80px;
+}
+
+.about-tournament p,
+.support-tournament p {
+    font-family: Merriweather;
+    font-size: 36px;
+    font-weight: 400;
+    line-height: 48.6px;
+    text-transform: uppercase;
+    display: inline;
+    margin: 0;
+}
+
+.red-text {
+    font-weight: 700;
+    color: rgba(245, 73, 50, 1);
+}
+
+.support-content {
+    display: flex;
+    margin-bottom: 17px;
+    justify-content: space-between;
+    align-items: center;
+}
```

```suggestion
.support-content {
    display: flex;
    margin-bottom: 17px;
    justify-content: space-between;
    align-items: center;
}
```

чтобы на маленьких экранах встало нормально, можно задать `flex-wrap: wrap` и будет магия 🪄 

ещё `gap: 8px` задать, чтобы был безопасный отступ между текстом и картинкой